### PR TITLE
Gin and pongo2 template engine

### PR DESCRIPTION
**Pull request by nguyenthenguyen**
Hope this help someone want to use Gin with [pongo2](https://github.com/flosch/pongo2) like me!

### CODE REVIEW

**Comment by manucorporat** [comment]
you didn't understand me: we can not submit a "demo" or example in contrib. but instead the pongo render (pong2-render/pongo2.go)

File: pong2-render/pongo2.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package main
```
Comment (manucorporat) [inline pong2-render/pongo2.go]: this should be renamed

**Comment by manucorporat** [comment]
I just notice that already exists a pongo2 integration  with Gin

**Comment by nguyenthenguyen** [comment]
Ok ! Thanks

**Pull request closed without merging**